RandomStudio/random-studio
# Blurred video background

- Generate bigger blurred background image at build time
- Generate the dominant color of this background
- Check the contrast between the dominant color and white text
- Add class to invert colors to controls if the contrast is not readable


Steps to test:

1. Open video focus page /video/[id]
2. You should see a blurred thumbnail filling the page
3. If the image is very light, the text will be black
4. Otherwise, it will be white
